pagopa/pn-frontend
# Refactor/PN-13434 - DigitalContactsCard Rework

## Short description
This PR includes the changes to `DigitalContactsCard` (and all the components which use it on both PF and PG webapps) to respect the latest figma specification. Moreover, the component has been renamed to `PnInfoCard` and moved into `pn-commons` package.

## How to test
Run pf and pg apps and temporarily change one of the card showed on the Contacts page (e.g. `EmailContactItem`) adding to the `PnInfoCard` the following props:

```
import ConstructionIcon from '@mui/icons-material/Construction';
import PowerSettingsNewIcon from '@mui/icons-material/PowerSettingsNew';
import { Button, Chip } from '@mui/material';
...
<PnInfoCard
...
subtitle={<Chip label="Da Attivare" color="secondary" sx={{ mb: 2 }} />}
actions={[
    <Button
        key="Gestisci"
        variant="naked"
        color="primary"
        startIcon={<ConstructionIcon />}
        onClick={() => console.log('Gestisci!')}
        sx={{ p: '10px 16px' }}
    >
        Gestisci
    </Button>,
    <Button
        key="Disattiva"
        variant="naked"
        color="error"
        startIcon={<PowerSettingsNewIcon />}
        onClick={() => console.log('Disattiva!')}
        sx={{ p: '10px 16px' }}
    >
        Disattiva
    </Button>,
]}
...
```

test the apps in both desktop and mobile mode to verify the layout is as expected.
